server/services/v1/database/query_runner.go

```diff
@@ -351,6 +355,8 @@ func (runner *DeleteQueryRunner) Run(ctx context.Context, tx transaction.Tx, ten
 			}
 		}
 
+		request.IncrementWrittenBytes(ctx, len(row.Data.RawData))
```

Comment:
If I delete a document of size 2KB, will this cause it to be counted as two write units?

Reply:
Yes, we will need to do two things here for the Delete:
- count the number of bytes read to fetch the data to be deleted if we need to read. If it is a blind delete, then there is nothing read, so zero bytes (read units consumed)
- count the number of documents deleted as 1 write each if we are doing individual deletes
- if it is a blind delete then it is just 1 operation